src/pages/HomePage/HomePage.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import HomePage from './HomePage';
+
+describe('HomePage', () => {
+  test('renders HomePage component', () => {
+    render(<HomePage value="Hello" />);
+
+    expect(screen.getByText('Hello'));
+  });
+});
```

Comment:
we need test to match home page. its throwing errors. add props matching home page and create some test based on them

Reply:
Added test cases.